README.md

```diff
@@ -80,11 +83,13 @@ const PeopleSearch = () = {
 - `pollInterval` - How often to re-request updated data. Pass 0 to disable polling (the default behaviour).
 - `payload` - A data object to send in the request. If we are performing a GET request, it is appended into the querystring (e.g. `?keywords=hello`). If it is a POST request it is sent in the request body as JSON.
 - `method` - Set the request type, either `get` or `post`. (defaults to `get`)
+- `changed`: A function to call if the data actually changed during the request.
```

Comment:
I think it would be good to state that no redraw will happen unless the data is changed, to make it explicit. Is there any sort of tradeoff for this? does it mean e.g. big data sets will be stored in memory twice (once by this tool and once by the calling app that uses this tool)? Does that need design consideration by whoever uses this tool?

Reply:
I've made the behaviour optional. If a change option is not specified, no comparison will be done. Also expanded in the README.